src/phoenix/server/api/schema.py

```diff
@@ -343,6 +347,42 @@ async def clear_project(self, info: Info[Context, None], id: GlobalID) -> Query:
                 cache.invalidate(ClearProjectSpansEvent(project_rowid=project_id))
         return Query()
 
+    @strawberry.mutation
+    async def create_dataset(
+        self,
+        info: Info[Context, None],
+        name: str,
+        description: Optional[str],
+        metadata: Optional[JSON],
```

Comment:
I think these need to be defaulted to `None`, otherwise, they will be required parameters.

```suggestion
        description: Optional[str] = None,
        metadata: Optional[JSON] = None,
```

Reply:
Oh interesting